gui/idact_app.py

```diff
@@ -1,8 +1,13 @@
 from PyQt5.QtWidgets import QMainWindow
 from PyQt5 import uic
+
+from gui.native_saver import NativeArgsSaver
 from gui.saver import ParameterSaver
 
 Ui_MainWindow, QtBaseClass = uic.loadUiType('../widgets_templates/gui.ui')
+Ui_AddNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/add-native.ui')
+Ui_RemoveNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/remove-native.ui')
+Ui_ShowNativeArgument, QtBaseClass = uic.loadUiType('../widgets_templates/show-native.ui')
```

Comment:
if argument QtBaseClass is not used it should be named as _QtBaseClass, now you override it each time